### PR TITLE
CSS & Test Optimisations, new pages Funding and Case Study

### PR DESCRIPTION
**Pull request by rosemcc**
- CSS optimisations
- Unit Test improvements
- Funding Page
- Case Study Page
- GZip Compression
- e2e tests not relying on specific content wording, and tests running on Jenkins
- Add some missing Contentful fields in front end
- Minor modifications to e2e config

### CODE REVIEW

File: research-hub-web/cypress/integration/homepage.e2e.js [hunk 1]
```diff
@@ -8,34 +8,35 @@ describe('ResearchHubs Homepage', () => {
...
     it('displays footer', () => {
-        cy.contains(`Centre for eResearch`);
+        cy.expect('footer .footer-content li:first-child').not.to.be.empty;
     })
 
-    it('clicking Sign In takes you to SSO', () => {
-        cy.get('#top-bar').contains('Sign In').click();
-        cy.contains('Or sign in with one of the following services');
-    })
+    // it('clicking Sign In takes you to SSO', () => {
```
Comment (robmint) [inline research-hub-web/cypress/integration/homepage.e2e.js:38]: Just to note these commented out tests are sorted out in PR #102

**Review by robmint — APPROVED** [review]

File: research-hub-web/cypress/integration/homepage.e2e.js (hunk 1 above)
Comment (rosemcc) [reply research-hub-web/cypress/integration/homepage.e2e.js:38]: PR #102 will be added to this PR

**Review by cameronmclean — APPROVED** [review]

**Comment by rosemcc** [comment]
PR #97 added to this PR

**Comment by rosemcc** [comment]
PR #104 added to this PR

**Pull request merged**